complete/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <script src="script.js"></script>
+  <meta name="description" content="Responsive Website">
+  <title>Responsive Website</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header class="header">
+    <div class="top-container">
+    <div class="logo"><img src="images/Logo.png" alt="logo"/></div>
+    <nav class="top-navbar">
+    <ul>
+      <li><a class="active" href="#home">Home</a></li>
+      <li><a href="#pages">Pages</a></li>
+      <li><a href="#features">Features</a></li>
+      <li><a href="#extensions">Extensions</a></li>
+      <li><a href="#tutorials">Tutorials</a></li>
+      <li><a href="#contact_us">Contact us</a></li>
+      </ul>
+      </nav>
+      <div class="top-nav-social">
+      <a href="#" class="fa fa-facebook"></a>
+      <a href="#" class="fa fa-twitter"></a>
+      <a href="#" class="fa fa-skype"></a>
+      <a href="#" class="fa fa-linkedin"></a>
+      </div>
+      </div>
+    <div class="header-text">
+        <h1>home</h1>
+        <h3>power by psdfreebies.com</h3>

```

Comment:
![letter-spacing](https://user-images.githubusercontent.com/2418147/42778742-67a941c4-893e-11e8-90b6-652697aef4c1.jpg)

The text `power by psdfreebies.com` is not like Photoshop, all spaced out and blue.
_HINT: there is a css property to space out letters in a text_